proposals/nnnn-AOA-multiplexing.md

```diff
@@ -0,0 +1,53 @@
+# AOA multiplexing similar to Android BT/SPP multiplexing
+
+* Proposal: [SDL-NNNN](NNNN-AOA-multiplexing.md)
+* Author: [Jennifer Hodges](https://github.com/jhodges55)
+* Status: Awaiting review
+* Impacted Platforms: [ Android ]
+
+
+## Introduction
+
+Add "multiplexing" to AOA transport. ![Overview](../assets/proposals/nnnn-AOA-multiplexing/nnnn-AOA-multiplexing_overview.png)
+This change introduces a better user experience as followings:
+- Multi navigation apps to be able to exchanged by user.
+- None navigation apps can select not only BT/SPP but AOA. 
+- In the future, projection will be applied to not only navigation but also other category application then, multi "projection apps" can be supported.
+
+## Motivation
+
+Currently, only one AOA app can connnect to SDL core because of AOA limitation. Before that, our assumption of AOA, only 1 navigation app can connect to SDL core. No more app is assumed. However, several navigation apps will come in the near future. And, projection functionality will be opened for not navi but other category apps which requires wide bandwidth such USB. So, for better and flexible UX, it should be valuable to support multiple app on AOA.
+ 
+## Proposed solution
+
+Implement multiplexing functionality to AOA transport of SDL Android library. This architecture is similar to existing BT/SPP multiplexing. However, in Android, SPP and AOA may exist in parallel. So, another multiplexing router for AOA should be added. If OEM using other protocol on AOA for none SDL service to be supported then, this architecture can be used for this purpose with customized SDL Android library which is special router service for other standard SDL apps.
```

Comment:
Please clarify/reword "If OEM using other protocol...for other standard SDL apps," sentence.